Running-Mastodon/Administration-guide.md

```diff
@@ -8,6 +8,7 @@ So, you have a working Mastodon instance... now what?
 The following rake task:
 
     RAILS_ENV=production bundle exec rails mastodon:make_admin USERNAME=alice
+    (or docker-compose run --rm web rails mastodon:make_admin USERNAME=alice)
```

Comment:
Actually, you're fine not using bundle exec inside Docker. Bundle exec is useful when there's a chance you have overlapping binaries/rubies. In docker, you only have the right stuff.